### DISCUSSION TITLE
Removed question.

### DISCUSSION THREAD

**Discussion by independentCog** [vim/vim#13355] [General]
I don't engage often on GitHub so I likely did something wrong without understanding it but I asked a question last night on here related to a warning message I got when compiling Vim from source and it seems to have been removed. I'd ask the question again but instead, I'd like some insight on what I did wrong and how I can more appropriately ask questions on this repo first. I wanted to learn Vim and in the process needed custom features to use it for my specific purposes.

I broke down the message, file, and line of code the warning was being generated from. I put my question in Q&A so was that the wrong topic for there? Any help or clarification would be appreciated on this. =)

**Comment by zeertzjq** [comment]
Because it has been fixed, and the question is still reachable at #13352.

**Reply by independentCog** [reply]
I closed this as soon as I saw it and just got the notification late. Dumb on my part. Thought I had asked the question in the wrong place and had it removed. New to discussion on Github so apologies for the misunderstanding.
